omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/RegisterPatientFragmentController.java

```diff
@@ -318,7 +321,7 @@ private void buildGroupObs(ConceptService conceptService, List<Obs> obsToCreate,
         }
     }
 
-    private void buildObs(ConceptService conceptService, List<Obs> obsToCreate, String conceptId, String[] parameterValues) throws ParseException {
+    private void buildObs(ConceptService conceptService, List<Obs> obsToCreate, String conceptId, String[] parameterValues,HttpServletRequest request,String name) throws ParseException {
```

Comment:
hi @reagan-meant   thanks for the works here,do we really need this around there HttpServletRequest request,checkout that

Reply:
Thanks @HerbertYiga for the review...I have changed the approach..see new PR